missions/2PzD_Template.VR/customization/loadouts/Finnish/loadoutFin39.sqf

```diff
@@ -0,0 +1,525 @@
+// Info: Finnish Army 1939 Loadouts
+// - For multi line definitions use \
+// - All loadouts listed are customised to correct equipment, so have fun with them
+// - ALWAYS ADD SHORT-RANGE RADIO FIRST
+// - On any random equipment definition remember to use Olsen_FW_FNC_AddItemRandom
+
+/* Loadouts
+
+    //Platoon HQ
+[this, F39_PC] call Olsen_FW_FNC_GearScript;          Platoon Commander
+[this, F39_PSgt] call Olsen_FW_FNC_GearScript;        Platoon Sergeant
+[this, F39_Mess] call Olsen_FW_FNC_GearScript;        Messenger
+[this, F39_MedP] call Olsen_FW_FNC_GearScript;        Medic
+[this, F39_RTO] call Olsen_FW_FNC_GearScript;         Radio Operator
+
+    //Squad
+[this, F39_SL] call Olsen_FW_FNC_GearScript;          Squad Leader
+[this, F39_TL] call Olsen_FW_FNC_GearScript;          Team Leader
+[this, F39_SMG] call Olsen_FW_FNC_GearScript;         Submachine Gunner
+[this, F39_Gren] call Olsen_FW_FNC_GearScript;        Grenadier
+[this, F39_MedS] call Olsen_FW_FNC_GearScript;        Assistant Medic
+
+[this, F39_Rif] call Olsen_FW_FNC_GearScript;         Rifleman
+[this, F39_MG] call Olsen_FW_FNC_GearScript;          Machine Gunner
+[this, F39_MGA] call Olsen_FW_FNC_GearScript;         MG Assistant
+
+    //Heavy Weapons Teams
+[this, F39_HMGTL] call Olsen_FW_FNC_GearScript;       HMG Team Leader
+[this, F39_HMGG] call Olsen_FW_FNC_GearScript;        HMG Gunner
+
+[this, F39_ATRTL] call Olsen_FW_FNC_GearScript;       AT Rifle Team Leader
+[this, F39_ATRG] call Olsen_FW_FNC_GearScript;        AT Rifle Gunner
+
+    //Vehicle Crew
+[this, F39_VCom] call Olsen_FW_FNC_GearScript;        Tank Commander
+[this, F39_VCrew] call Olsen_FW_FNC_GearScript;       Tank Crew
+*/
+
+//======================== Definitions ========================
+
+//For Riflemen
+#define F39_Weapon_Rifleman \
+        [ \
+            [ \
+                [Fin_Vest_Mosin], \
+                [Fin_Mag_Mos,1], \
+                [Fin_Weap_MosM27], \
+                [Fin_Mag_Mos,12,"vest"] \
+            ],[40], \
```

Comment:
40/40 chance? Shouldn't this be 50/50? 

(Note that this is functionally the same, as far as the code cares but it looks better if it's set to 50/50)

Reply:
It's 50/50 now, I forgot to change that when I copied that over from the 41 loadout which I did first and contains 3 weapons